Make.inc

```diff
@@ -132,6 +132,10 @@ else
 JULIA_COMMIT := $(JULIA_VERSION)
 endif
 
+# Increase these every release when breaking ABI in a backward-incompatible way
```

Comment:
https://autotools.io/libtool/version.html and http://plan99.net/~mike/writing-shared-libraries.html discuss library version in some detail